Makefile

```diff
@@ -48,7 +48,7 @@ RELEASE_CPP_LIB_OBJECTS := $(subst build/release-cpp/main.o,,$(RELEASE_CPP_OBJEC
 all: release
 
 clean:
-	@rm -rf build wren wrend libwren libwrend
+	@rm -rf build wren wrend libwren* libwrend*
```

Comment:
Seeing any kind of `*` after `rm -rf` gives me the heebie-jeebies, but I agree that the current Makefile is wrong. How about just listing all four names explicitly?